datajunction-clients/python/README.md

```diff
@@ -130,12 +130,23 @@ List of all available DJ client methods:
         filters: Optional[List[str]],
         engine_name: Optional[str],
         engine_version: Optional[str])
+  - node_sql( node_name: str,
```

Comment:
Can this also propagate `limit` as a parameter? That will be important since in a lot of cases these nodes are very large (since this is now moving away from just metrics, which are more limited in size, but supporting general nodes, and often the transforms are very large).

Reply:
there's no code for `limit` as a parameter in `client.py` so maybe that should be a separate PR?

Reply:
Sure, yeah, separate PR sounds good